### PR TITLE
Add stream with no funds to claim

### LINKED ISSUE #3856: Expenditures: As a user I can see a active stream with no funds to claim
[Provided by Adrian via Contractors project management platform]

## Description
User has to be able to see a active stream with no funds to claim. "Paid to date" and "Available to claim" may vary depends on how many currencies are used in funding source.

## Acceptance Criteria

- an option to see an "Ended" state
- an option to see "Claimed" button without option to click it
- an option to see "Paid to date" funds and currencies
- an option to see "Available to claim" funds and currencies (0 funds)
- an option to see a "passed" state below linked motions

## Design

- [Figma Link - Stream with no funds to claim](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5544%3A601778)

Until - popover 8

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

This PR adds active stream with no funds to claim. UI only. Values and functions are mocks.
After clicking "Claim funds" button:
- "Available to claim" funds  are equal to 0,
-  button displays "Claimed" text and is disabled,
-  "Ended" state is visible,
- "passed" state below linked motions is displayed.
The rest of the states (failed, cancelled, ...) will be added in other PRs.

Screenshot 2022-09-22 at 14 53 14

[Figma link](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=5544%3A601778)

Resolves #3856

### CODE REVIEW

**Review by willm30 — COMMENTED** [review]
Looks good. 

Couple of things: 
1. I know the data is mocked, but why is the number showing up twice?
2. The "passed" state shows up for me before I've clicked "Claimed"

claim

**Comment by agnieszkajarosikloj** [comment]
@willm30 
1. I changed mocked data, because token was incorrect. And I fixed displaying amount, when token was not found.
2. The "passed" status refers to the motion to create streaming payment (after clicking "Start stream" button).

**Review by willm30 — APPROVED** [review]
Nice one 👍

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx [hunk 1]
```diff
@@ -288,13 +306,14 @@ const StreamingStagesLocked = ({
...
+                    return null;
+                  }
 
                   return (
                     <span
                       className={classNames(styles.value, {
-                        [styles.marginBottom]:
-                          availableToClaim.length > 1 &&
-                          index !== availableToClaim.length - 1,
+                        [styles.marginTop]: index > 0,
```
Comment (jakubcolony) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:316]: Similarly to my question in #3888, wouldn't it make sense for that to be done in CSS?

**Review by jakubcolony — APPROVED** [review]
Looks good to me!

image

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css [hunk 2]
```diff
@@ -130,6 +135,6 @@
   align-items: flex-start;
 }
 
-.marginBottom {
-  margin-bottom: 24px;
+.marginTop {
```
Comment (chinins) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css:138]: We try not to use class names like these. Please, provide a more meaningful name (e.g. container of sorts).

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx [hunk 3]
```diff
@@ -57,6 +57,14 @@ const MSG = defineMessages({
     id: `dashboard.ExpenditurePage.Stages.StreamingStages.StreamingStagesLocked.claimFunds`,
     defaultMessage: 'Claim funds',
   },
+  ended: {
+    id: `dashboard.ExpenditurePage.Stages.StreamingStage.StreamingStagesLocked.ended`,
+    defaultMessage: 'Ended',
+  },
+  claimed: {
+    id: `dashboard.ExpenditurePage.Stages.StreamingStage.StreamingStagesLocked.ended`,
```
Comment (chinins) [inline src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:65]: wrong id - should be `claimed` at the end

File: src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx (hunk 1 above)
Comment (chinins) [reply src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx:316]: I agree, I think you can use css selectors for that.

**Review by chinins — COMMENTED** [review]
All good, just please update the css as per Jakub's and my comment. Please, let me know once done & i'll re-review straight away.

**Pull request merged**